src/components/Announcements/AnnouncementsList.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="scroll-container announcements-container" v-bind:style="{ 'maxHeight': '20rem' }">
```

Comment:
I want to change this style binding to something more extensible. Like maybe a scroll container component similar to access component where you just wrap it around whatever content you want to scroll, pass a max height in as a prop, and your inner content will just get slotted in. But because we're sort of running low on time I figured this wasn't high priority and am keeping it on ice for if there is time or if somebody thinks it's important.